v2/internal/internalgauntlettest/thrift.go

```diff
@@ -37,7 +37,8 @@ import (
 type thriftHandler struct{}
 
 func thriftProcedures() []yarpc.TransportProcedure {
-	return echoserver.New(thriftHandler{})
+	procedures, _ := yarpc.EncodingToTransportProcedures(echoserver.New(thriftHandler{}))
```

Comment:
Let's handle this error. If we want to keep the return signature the same, we can take `*testing.T` as a parameter and enforce the requirement here.

```
func thriftProcedures(t *testing.T) []yarpc.TransportProcedure {
    procedures, err := yarpc.EncodingToTransportProcedures(echoserver.New(thriftHandler{}))
    require.NoError(t, err)
    return procedures
}
```

Reply:
Kris mentioned in his review that we should consider dropping the error in `EncodingToTransportProcedures(..)`. Let me know your thoughts on it.